ui/lib/kv/addon/routes/secret/patch.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Route from '@ember/routing/route';
+import AdapterError from '@ember-data/adapter/error';
+import { breadcrumbsForSecret } from 'kv/utils/kv-breadcrumbs';
+import { service } from '@ember/service';
+export default class SecretPatch extends Route {
+  @service version;
+
+  beforeModel() {
+    if (!this.version.isEnterprise) {
+      throw this.generateError({ message: 'Patching a KV v2 secret is only available on Vault Enterprise.' });
+    }
+  }
```

Comment:
Throws if users navigate directly to the `/patch` url
<img width="858" alt="Screenshot 2024-08-26 at 5 54 45 PM" src="https://github.com/user-attachments/assets/8abe0ddb-b8d0-4157-80f4-e104b3661cf0">